### PR TITLE
feat: expose Transformer

### PR DESCRIPTION
**Pull request by lpatiny**

### CODE REVIEW

**Comment by lpatiny** [comment]
@targos I have again the problem of the build that does not contain `Transformer`.

I have installed the last version of 'GWT' and I'm using the last version of actelion java code. Is this correct ? My config file is:

```json
{
  "gwt": "/Users/lpatiny/gwt/gwt-2.10.0",
  "openchemlib": "/Users/lpatiny/git/actelion/openchemlib/src"
}

```

Trying to compile `npm run build-debug-full` I get 442 errors in the log file like:

```
   Tracing compile failure path for type 'com.actelion.research.util.datamodel.IntArray'
      Errors in 'file:/Users/lpatiny/git/cheminfo/openchemlib-js/src/com/actelion/research/gwt/chemlib/com/actelion/research/util/datamodel/IntArray.java'
         Line 458: No source code is available for type java.util.StringTokenizer; did you forget to inherit a required module?
      Checked 1 dependencies for errors.
   Tracing compile failure path for type 'com.google.gwt.validation.client.impl.GwtValidationContext'
      Errors in 'com/google/gwt/validation/client/impl/GwtValidationContext.java'
         Line 25: The import javax.validation.metadata cannot be resolved
         Line 116: TraversableResolver cannot be resolved to a type
         Line 136: ConstraintDescriptor cannot be resolved to a type
         Line 70: TraversableResolver cannot be resolved to a type
```

**Comment by targos** [comment]
You have to look for `[ERROR]` in the logs (not sure about the syntax, but it contains the brackets). Many errors are not problematic (because the code is not used in the end)

**Comment by targos** [comment]
You can also search for the name of the class that you add/modify.

In this case, here is the relevant error:

```
   Tracing compile failure path for type 'com.actelion.research.gwt.core.JSTransformer'
      Errors in 'file:/home/runner/work/openchemlib-js/openchemlib-js/src/com/actelion/research/gwt/core/JSTransformer.java'
         Line 7: JsType cannot be resolved to a type
      Checked 1 dependencies for errors.
```

It's just missing an import for the `JsType` decorator. In other files we do `import jsinterop.annotations.*;`

File: src/com/actelion/research/gwt/minimal/JSReaction.java [hunk 1]
```diff
@@ -41,6 +41,10 @@ public native Object getOCL()
     return $wnd.OCL;
   }-*/;
 
+  public Reaction getReaction() {
```
Comment (targos) [inline src/com/actelion/research/gwt/minimal/JSReaction.java:44]: should be `@JsIgnore` as this method doesn't make sense in JS

**Pull request merged**